packages/core/src/components/panel-stack/panel-stack.scss

```diff
@@ -0,0 +1,99 @@
+// Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+
+@import "../../common/variables";
+@import "~@blueprintjs/core/src/common/react-transition";
+
+.#{$ns}-panel-stack {
+  position: relative;
+  overflow: hidden;
+
+  .push-#{$ns}-panel-stack-transition-enter {
+    transform: translateX(100%);
+  }
+
+  .push-#{$ns}-panel-stack-transition-enter-active {
+    transform: translateX(0%);
+    transition: all 300ms ease-in;
+  }
+
+  .push-#{$ns}-panel-stack-transition-exit {
+    transform: translateX(0%);
+  }
+
+  .push-#{$ns}-panel-stack-transition-exit-active {
+    transform: translateX(-100%);
+    transition: all 300ms ease-in;
+  }
+
+  .pop-#{$ns}-panel-stack-transition-enter {
+    transform: translateX(-100%);
+  }
+
+  .pop-#{$ns}-panel-stack-transition-enter-active {
+    transform: translateX(0%);
+    transition: all 300ms ease-in;
+  }
+
+  .pop-#{$ns}-panel-stack-transition-exit {
+    transform: translateX(0%);
+  }
+
+  .pop-#{$ns}-panel-stack-transition-exit-active {
+    transform: translateX(100%);
+    transition: all 300ms ease-in;
+  }
+}
+
+.#{$ns}-panel-stack-header {
+  display: flex;
+  flex-shrink: 0;
+  z-index: $pt-z-index-content + 2;
+  box-shadow: 0 1px $pt-divider-black;
+  height: 30px;
+
+  .#{$ns}-panel-stack-header-back {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    padding: 0 5px;
+    text-transform: uppercase;
+    letter-spacing: 0.5px;
```

Comment:
whoa, prefer not messing with text-transform or letter-spacing, they're too design-specific